tests/test_xtai_calendar.py

```diff
@@ -64,4 +64,6 @@ def non_holidays_sample(self):
             "2015-06-22",  # Dragon Boat Festival on Saturday, Jun 20th.
             "2015-09-25",  # Mid-Autumn Festival on Sunday, Sep 27th.
             "2015-10-12",  # National Day on Saturday, Oct 10th.
+            "2024-09-16",  # Mid-Autumn Festival on Tuesday, Sep 17th.
```

Comment:
For each of those holidays that ceased to observe the extra monday / extra friday from 2024, could you please include the next date that would otherwise have been a holiday. Thanks